core-web/libs/sdk/client/src/lib/editor/models/client.model.ts

```diff
@@ -42,11 +42,14 @@ export enum CUSTOMER_ACTIONS {
      * Tell the editor to save inline edited contentlet
      */
     UPDATE_CONTENTLET_INLINE_EDITING = 'update-contentlet-inline-editing',
-
     /**
      * Tell the editor to trigger a menu reorder
      */
     REORDER_MENU = 'reorder-menu',
+    /**
+     * Tell the editor to send the page info to iframe
+     */
+    GET_PAGE_INFO = 'get-page-info',
```

Comment:
```suggestion
    GET_PAGE_DATA = 'get-page-data',
```

The page API inside have the page info (under .page) which is the page contentlet information, all the fields etc...

This data here is way more than just the contentlet... I am proposing `data` but it could be something else, just want to make very clear that is everything, layout, containers etcs.

Reply:
I agree, changed!